src/main/resources/db/migration/V2023_05_05_1015__Database_initialization.sql

```diff
@@ -0,0 +1,8 @@
+create table book(
+    id serial,
+    title varchar,
```

Comment:
I guess, there are several fields here, which cannot be nullable. Please, add constraints